content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,567 @@ import Tooltip from "shared/components/ui/Tooltip";
 
-<WarningBox>
-Info: Only Kubernetes versions 1.19 and above is supported
-</WarningBox>
+Spectro Cloud offers Cloud Native Computing Foundation (CNCF) Kubernetes as a core infrastructure pack in Palette. We release Kubernetes updates as follows:
+
+<br />
+
+- Major versions are assessed based on the extent of changes.
+- Minor versions are provided within eight weeks of a new Kubernetes release.
+- Patch versions are provided within four weeks of a new Kubernetes release.
 
-## Supported Kubernetes versions
+In the period of time following a Kubernetes release, the Kubernetes pack undergoes rigorous vulnerability scans and penetration testing. 
 
-<InfoBox>
+A minor Kubernetes version will be deprecated in Palette when the Kubernetes community announces the version is entering End of Life (EOL). The deprecated version will remain in that state for three months and will be disabled after a quarter and deleted from Palette in the following quarter.
 
-Versions supported in the latest [release](/release-notes/) are highlighted.
+<br />
+
+## Versions Supported
 
-</InfoBox>
 
 <Tabs>
 
-<Tabs.TabPane tab="1.19.x" key="k8s_v1.19">
-
-- **1.19.16**
-- **1.19.15**
-- **1.19.14**
-- **1.19.13**
-- **1.19.12**
-- **1.19.11**
-- **1.19.10**
-- **1.19.9**
-- **1.19.8**
-- **1.19.7**
-- **1.19.6**
-- **1.19.5**
-- **1.19.4**
-- **1.19.3**
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
 
-</Tabs.TabPane>
+## Prerequisites
 
-<Tabs.TabPane tab="1.20.x" key="k8s_v1.20">
-
-- **1.20.12**
-- **1.20.11**
-- **1.20.10**
-- **1.20.9**
-- **1.20.8**
-- **1.20.7**
-- **1.20.6**
-- **1.20.5**
-- **1.20.4**
-- **1.20.2**
-- **1.20.1**
-- **1.20.0**
+- A minimum of 4 CPU and 4GB Memory.
 
-</Tabs.TabPane>
+- Operating System (OS) dependencies as listed in the table.
 
-<Tabs.TabPane tab="1.21.x" key="k8s_v1.21">
+| OS Distribution | OS Version | Supports Kubernetes 1.26.x |
+|---------------|------------|----------------------------|
+| CentOS        | 7.7        | ❌                         |
+| Ubuntu        | 22.04      | ✅                         |
+| Ubuntu        | 20.04      | ❌                         |
+| Ubuntu        | 18.04      | ❌                         |
 
-- **1.21.10**
-- **1.21.6**
-- **1.21.5**
-- **1.21.4**
-- **1.21.3**
-- **1.21.2**
-- **1.21.1**
-- **1.21.0**
 
-</Tabs.TabPane>
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| ``pack:k8sHardening``  | Flag to decide if Kubernetes hardening should be applied. Default: ``True``. When set to ``True``, additional flags configured in `kubeadmconfig` will be honored and will be set to the corresponding components.|
+| ``pack:podCIDR`` | The CIDR range for Pods in cluster. This should match the networking layer property ``calicoNetworkCIDR``. Default: `192.168.0.0/16`|
```

Comment:
calicoNetworkCIDR - this will be present only if calico is used as CNI.  You can say that - "This should match the corresponding networking layer property. Eg: calicoNetworkCIDR. Same applies to other k8s versions below as well.
"

Reply:
Good catch @vishwanaths. Removed mention of calicoNetworkCIDR.